packages/nextjs/test/buildProcess/testApp/yarn.lock

```diff
@@ -108,14 +108,14 @@
     estree-walker "^2.0.1"
     picomatch "^2.2.2"
 
-"@sentry/browser@7.20.0":
-  version "7.20.0"
-  resolved "https://registry.yarnpkg.com/@sentry/browser/-/browser-7.20.0.tgz#e917cab48d3bba2fe8a2e3a8987b0bd9b9149db8"
-  integrity sha512-L84CdB7DPQ2ohVcWh/KivemndWSZyXRvBZBr+tHFlQchzcaZZ/8lIPvjwvb8OJhzhecDq6JCAyUxaZwyItdyAg==
-  dependencies:
-    "@sentry/core" "7.20.0"
-    "@sentry/types" "7.20.0"
-    "@sentry/utils" "7.20.0"
+"@sentry/browser@7.20.1":
```

Comment:
This is the test app in the build process tests, not our main yarn.lock. I think I get why Kamil set his integration test app to not use a lockfile, because this is going to happen every time we bump a version, but that's handlable in a separate PR. I can pull this bit out of this PR if you feel strongly about it.

Reply:
Ah I am dumb sorry